hedera-node/hedera-app/src/xtest/java/token/TokenUpdateXTest.java

```diff
@@ -0,0 +1,151 @@
+                                .kycKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .wipeKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .supplyKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .feeScheduleKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .pauseKey(IMMUTABILITY_SENTINEL_KEY))
+                        .build(),
+                OK);
+        handleAndCommitSingleTransaction(
+                component.tokenUpdateHandler(),
+                TransactionBody.newBuilder()
+                        .transactionID(TransactionID.newBuilder()
+                                .accountID(DEFAULT_PAYER_ID)
+                                .build())
+                        .tokenUpdate(TokenUpdateTransactionBody.newBuilder()
+                                .token(idOfNamedToken(TOKEN_ID))
+                                .metadataKey(IMMUTABILITY_SENTINEL_KEY))
```

Comment:
In the HIP there are **no** specifications about the `metadataKey`, hence all 
`metadataKey` mentions are irrelevant and should be removed. This includes all verification mentions and changes

Reply:
Removed, thanks